README.rst

```diff
@@ -1,20 +1,48 @@
-vivarium_data_requests
 ======================
```

Comment:
I don't know if it renders differently, but we only have an underline in the other repos

Reply:
I think it's just a signal that this is the top level header since rst does not care about what punctuation you use.  It doesn't render any differently though (P.S you can use the button in the top left to render the rich diff for stuff like that).